Tests/Functional/FailureMessagesTestCase/main.swift

```diff
@@ -133,13 +142,20 @@ class FailureMessagesTestCase: XCTestCase {
         XCTAssertNil("helloworld", "message", file: "test.swift")
     }
 
+// CHECK: Test Case 'FailureMessagesTestCase.testAssertNotEqualValues' started at \d+:\d+:\d+\.\d+
+// CHECK: test.swift:[[@LINE+3]]: error: FailureMessagesTestCase.testAssertNotEqualValues : XCTAssertNotEqual failed: \("1"\) is equal to \("1"\) - message
+// CHECK: Test Case 'FailureMessagesTestCase.testAssertNotEqualValues' failed \(\d+\.\d+ seconds\).
+    func testAssertNotEqualValues() {
+        XCTAssertNotEqual(1, 1, "message", file: "test.swift")
+    }
+
 // CHECK: Test Case 'FailureMessagesTestCase.testAssertNotEqualOptionals' started at \d+:\d+:\d+\.\d+
 // CHECK: test.swift:[[@LINE+3]]: error: FailureMessagesTestCase.testAssertNotEqualOptionals : XCTAssertNotEqual failed: \("Optional\(1\)"\) is equal to \("Optional\(1\)"\) - message
 // CHECK: Test Case 'FailureMessagesTestCase.testAssertNotEqualOptionals' failed \(\d+\.\d+ seconds\).
     func testAssertNotEqualOptionals() {
-        XCTAssertNotEqual(1, 1, "message", file: "test.swift")
+        XCTAssertNotEqual(Optional(1), Optional(1), "message", file: "test.swift")
     }
-
+    
```

Comment:
nit-pick: Same as above, looks like you added some extra whitespace here. Can you remove it?